extension/ezoe/design/standard/javascript/plugins/eztable/editor_plugin.js

```diff
@@ -971,33 +1015,36 @@
 
 							do {
 								// Text node
-								if (node.nodeType == 3 && tinymce.trim(node.nodeValue).length != 0) {
-									if (start)
-										rng.setStart(node, 0);
-									else
-										rng.setEnd(node, node.nodeValue.length);
+							if (node.nodeType == 3 && tinymce.trim(node.nodeValue).length !== 0) {
+                                    if (start) {
+                                        rng.setStart(node, 0);
+                                    } else {
+                                        rng.setEnd(node, node.nodeValue.length);
+                                    }
 
 									return;
 								}
 
 								// BR element
 								if (node.nodeName == 'BR') {
-									if (start)
-										rng.setStartBefore(node);
-									else
-										rng.setEndBefore(node);
+                                    if (start) {
```

Comment:
Yes, my fault. I need to merge last one patch (https://github.com/adamwojs/tinymce/commit/b85c2cda8d7b44235f1b51bb907e543b7f667b79) so I'm still working on this PR :-)